crowbar_framework/app/models/service_object.rb

```diff
@@ -1631,9 +1631,29 @@ def equal_or_match_version(value, maybe_regexp)
       end
     end
 
-    cmp = lambda do |op, version|
-      (to_version.call(value).zip to_version.call(version)).all? do |v1, v2|
-        v1.send(op, v2)
+    cmp = lambda do |version1, version2|
+      version1.zip version2.each do |v|
```

Comment:
I find this line hard to parse without parenthesis. Is this the same as `version1.zip(version2).each`? If yes, could you rewrite it like that?